src/main/java/org/apache/maven/plugin/compiler/AbstractCompilerMojo.java

```diff
@@ -483,7 +483,19 @@
     private List<String> fileExtensions;
 
     /**
-     * to enable/disable incrementation compilation feature
+     * <p>to enable/disable incrementation compilation feature.</p>
+     * <p>This leads to two different modes depending on the underlining compiler. Default javac compiler does the
+     * following:</p>
+     * <ul>
+     * <li>true <strong>(default)</strong> in this mode the compiler plugin determines whether any JAR files the
+     * current module depends on have been changed in the current build run, or any source file was added, removed or
+     * changed since the last compilation. If this is the case, the compiler plugin recompiles all sources</li>
+     * <li>false <strong>(not recommended)</strong> this only compiles source files which are newer than their
+     * corresponding class files. That is which have been changed since the last compilation. This does not
```

Comment:
That is...
is a sentence fragment

Reply:
Thanks for fixing my terrible english writing! It is much appreciated :hugs: